lib/auth/storage/storage.go

```diff
@@ -233,6 +235,31 @@ func (p *ProcessStorage) WriteTeleportVersion(ctx context.Context, version *semv
 	return nil
 }
 
+func (p *ProcessStorage) rdpLicenseKey(majorVersion, minorVersion uint16, issuer, company, productID string) backend.Key {
+	return backend.NewKey("rdplicense", issuer, strconv.Itoa(int(majorVersion)), strconv.Itoa(int(minorVersion)), company, productID)
```

Comment:
Are we ever going to list the licenses? If so, are we more likely to want to list them by issuer major minor company productid, or some other order?

Reply:
Order of the keys don't really matter as we don't ever need to list them. The only operation we need to support is "does a license exist for these specific values?"